code-challenges.test.js

```diff
@@ -21,11 +33,31 @@ const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 // b) Create the function that makes the test pass.
+// Pseudo: create a function called fibSequence that takes in a number greater than 2 and returns the an array of the same length containing the numbers following the rules of the Fibonacci sequence.
+// Within the function create an array called fibArray that contains the values [1,1]. Then create a for loop that iterates through fibArray while adding the current number and the previous number together but only when the index is at 2. The results will then be pushed into fibArray. The function will then return the desired Fibonacci sequence in fibArray.
+// const fibSequence = (number) => {
+//   let fibArray = [1, 1]
+//   for (let i = 2; i < number; i++) {
+//     fibArray.push(fibArray[i-1] + fibArray[i-2]);
+//   }
+//   return fibArray;
+// }
+
+
 
 // --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
 // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
 
 // a) Create a test with expect statements for each of the variables provided.
```

Comment:
A great way to get some brownie points in these tests are hoisting your given variables into your tests.
Example: 
```
describe("sortObject", () => {
  it(" takes in an object and returns an array of the values sorted from least to 
  greatest.", () => {
    const studyMinutesWeek1 = {
    sunday: 90,
    monday: 30,
    tuesday: 20,
    wednesday: 15,
    thursday: 30,
    friday: 15,
    saturday: 60
  }
  const studyMinutesWeek2 = {
    sunday: 100,
    monday: 10,
    tuesday: 45,
    wednesday: 60,
    thursday: 20,
    friday: 15,
    saturday: 65
  }
  expect(sortObject(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90])
  expect(sortObject(studyMinutesWeek2)).toEqual([10, 15, 20, 45, 60, 65, 100])
 })
})
```

So that is sits inside of your test after your it statement and before your expect statements. Also, you comment out your code after to run the console logs, but if your function that you create in part two works correctly it will return a passed test when you test it again because it will test the corresponding naming convention that matches to your function. Again, not a big one but definitely sets you a part from other junior devs